app/views/local_transactions/_fields.html.erb

```diff
@@ -48,6 +48,41 @@
           <%= f.text_area :need_to_know, rows: 4, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
         </span>
       </div>
+
+      <% administrations = {
+        "Scotland" => :scotland_availability,
+        "Wales" => :wales_availability,
+        "Northern Ireland" => :northern_ireland_availability
+      } %>
+      <% administrations.each do |title, field_name| %>
+        <%= f.fields_for field_name do |f| %>
+          <h3 class="govuk-heading-s"><%= f.label :type, title %></h3>
```

Comment:
`govuk-heading-s` is a class from `govuk-frontend`, but I don't think Publisher uses that. Does this class do anything?

Reply:
Thanks Andy - updated to `<p>` with `h4` class.